operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -156,15 +209,15 @@ When triaging a bug report, consider the following:
 
 **Q: What is community-daily.mattermost kept on?**
```

Comment:
Below is the current documented info on the differences between community servers, not sure if we'd want to add more details:

**Q: What is community.mattermost kept on?**

  - A: community.mattermost is kept on the most recent Mattermost Cloud release.

**Q: What is community-release.mattermost kept on?**

  - A: community-release.mattermost is kept on the currently in-progress Mattermost Cloud release.

**Q: What is community-daily.mattermost kept on?**

  - A: Normally on ``master`` branch and it updates once a day.

Reply:
@jwilander Do you think there is any other information that could be added here to help devs understand the differences between the community servers better?